.distignore

```diff
@@ -0,0 +1,14 @@
+/.github
+/.git
+/.wordpress-org
+/.vscode
+/node_modules
+.distignore
+.gitignore
+.nvmrc
+composer.json
+composer.lock
+package-lock.json
+package.json
+phpcs.xml
+readme.md
```

Comment:
Since we are not (currently) using the Composer autoloader, we should probably ignore the `/vendor` directory too. Or start using the Composer autoloader (there are historical reasons that we don't yet).

Reply:
Added /vendor and gitattributes to distignore